DesktopUI2/DesktopUI2/ViewModels/HomeViewModel.cs

```diff
@@ -202,10 +202,6 @@ private async Task GetStreams()
 
           streams.AddRange(result.Select(x => new StreamAccountWrapper(x, account.Account)));
         }
-        catch (OperationCanceledException)
-        {
-          return;
```

Comment:
The purpose of this return was to prevent user cancellation (through the cancellation token) from creating an error popup.

Testing your branch, it seems this is now broken.

https://github.com/specklesystems/speckle-sharp/assets/45512892/65425c7a-68ee-4ea4-902d-08a3cc3a24e2

Reply:
I've just tested the following code, thinking it could be a possible fix.
And while it does catch some of the cancellations, it looks like some are still originating from other cancellation tokens. Which is a little concerning.
```cs
        catch (OperationCanceledException) when (StreamGetCancelTokenSource.Token.IsCancellationRequested)
        {
          return;
        }
 ```
Some more investigation is required.

Reply:
Ahh damn, I didn't think about the search and that it can be canceled!
If we change the `return` with a `continue` would it work?

Reply:
That seems to work, shall I commit to your branch?